packages/core/src/app/customer/Customer.tsx

```diff
@@ -419,9 +428,11 @@ class Customer extends Component<CustomerProps & WithCheckoutCustomerProps, Cust
             executePaymentMethodCheckout,
             onContinueAsGuest = noop,
             providerWithCustomCheckout,
+            emitAnalyticsEvent,
         } = this.props;
 
         if (providerWithCustomCheckout) {
+            emitAnalyticsEvent("Account recognition");
```

Comment:
should this have additional field that can tell us the results of recognition? ie. Bolt shopper = true and guest shopper = false

Reply:
unfortunately, the line beneath this does not return whether or not the account exists. the bolt modal and account check all happens on a separate sdk that's not accessible on this checkout script